### PR TITLE
Fixed typos in documentation

### PR DESCRIPTION
**Pull request by Meuko**
My apologies, didn't think I'd read on today, but I did and found some more typos.

### CODE REVIEW

File: docs/start/types.extend.md [hunk 1]
```diff
@@ -212,4 +212,4 @@ Always look at customization and understand the impacts, replicating these chang
 
 ## Custom RPC
 
-In addition to customizing modules and types on your node, you can also add custom RPC definitions. Like the type definitions in ths section, [these can be defined and passed to the API](rpc.custom.md) for decoration.
+In addition to customizing your node's modules and types, you can also add custom RPC definitions. Like the type definitions in this section, [these can be defined and passed to the API](rpc.custom.md) for decoration.
```
Comment (Meuko) [inline docs/start/types.extend.md:215]: Please double check this, think I derped here.

File: docs/start/typescript.user.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # TypeScript user generated
 
-In the previous section we looked at the TypeScript definitions that are available and is generated from both the chain and definitions. Here we will expand upon the use of the infrastructure created to define types as part of the `@polkadot/types` library and see how to use them to generate your own definitions and chain types.
+In the previous section we looked at the TypeScript definitions that are available and are generated from both the chain and definitions. Here we will expand upon the use of the infrastructure created to define types as part of the `@polkadot/types` library and see how to use them to generate your own definitions and chain types.
```
Comment (Meuko) [inline docs/start/typescript.user.md:3]: nitpick, but the second are can be removed.

**Review by Meuko — COMMENTED** [review]
Please review these.

**Review by jacogr — APPROVED** [review]
lgtm. Thank you very much!

**Pull request merged**